ci/config.yml

```diff
@@ -0,0 +1,85 @@
+# Java Gradle CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: springcloudservices/spring-cloud-app-broker-ci:v1
```

Comment:
As Scott pointed out, might be best to create a new account that it's just for OSS rather than mixing. In case we want to grant access to members outside of the team. Happy to have it in a separate chore.

Reply:
There is a `springcloud` repo on Docker Hub. We should be able to get access to that. https://hub.docker.com/r/springcloud/